### ISSUE TITLE
Submit activities does not include the  the ticket name field

### ISSUE THREAD

**Issue by chaps** [ucoria-itexico/nonova#26]
Submit activity does not implement the ticket name field/Value.

**Comment by radiotaiso** [comment]
Yep, can be fixed by adding the ticket field/value to the construction of the object right [here](https://github.com/UpsetPelican/nonova/blob/master/activity.py)

**Comment by radiotaiso** [comment]
And perhaps a few extra tweaks elsewhere, but that's the baseline
